docs/source/toy/stochastic_logistic_model.rst

```diff
@@ -0,0 +1,7 @@
+*************************
+Stochastic Logistic Model
```

Comment:
Seems to have multiple names? Which one are we using?

Reply:
It's logistic.